src/MOI_wrapper.jl

```diff
@@ -5,6 +5,16 @@ const VI = MOI.VariableIndex
 
 const MOIU = MOI.Utilities
 
+"""
+    ADMMIterations()
```

Comment:
Maybe `NumberOfADMMIterations` to be consistent with other MOI attributes ?

Reply:
As a counter point,
http://www.juliaopt.org/MathOptInterface.jl/stable/apireference/#MathOptInterface.SimplexIterations
http://www.juliaopt.org/MathOptInterface.jl/stable/apireference/#MathOptInterface.BarrierIterations

(Maybe we should remove or refactor those attributes. They're very specific to MIP solvers.)

Reply:
Ok, then `ADMMIterations` is better indeed :)